### DISCUSSION TITLE
Difference between Horus V1 and V2?

### DISCUSSION THREAD

**Discussion by whallmann** [mikaelnousiainen/RS41ng#78] [General]
Some guys around asking what are the benefits for the newer V2?  the TX time is 4 seconds - 1 second longer, so there must be something for that it is good for. Who likes to explain the differences? Thanks.

**Comment by mikaelnousiainen** [comment]
@whallmann Please see https://github.com/projecthorus/horusdemodlib/wiki/4-Packet-Format-Details for differences between Horus V1 and V2. In V2 there are more payload IDs and space for custom data (custom fields), e.g. for additional sensors.

**Reply by mikaelnousiainen** [reply]
Additionally, Horus V1 has been deprecated because the 256 payload IDs it supported were not enough, so new IDs are not allocated anymore.

**Comment by whallmann** [comment]
Ok, last info i read already. I asked because there are still some peoples
here who likes the shorter tx time to spare some mAmps, hi hi. Most User do
not use extra Sensors but all transmit the overhead in V2.  So Life is 😄
thanks
Wolf

Mikael Nousiainen ***@***.***> schrieb am Mo., 18. Dez. 2023,
22:11:
